share/completions/killall.fish

```diff
@@ -22,7 +22,11 @@ if test (uname) != 'SunOS'
 		end
 	end
 
-	complete -c killall -xa '(__fish_complete_proc)'
+    if test $USER = root
+        complete -c killall -xa '(__fish_complete_proc)'
+    else
+       complete -c killall -xa "(ps ua | awk '{print \$11}' | sed '1d;s/.*\///;/^(/d')"
```

Comment:
Shouldn't that be `ps u`? The `a` option lists all processes with a tty, including things like shells owned by root.